sdk/monitor/azquery/README.md

```diff
@@ -33,21 +31,21 @@ The clients default to the Azure Public Cloud. See the [cloud][cloud_documentati
 
 #### Create a logs client
 
-Example logs client: [link][example_logs_client]
+Example [logs client][example_logs_client]
```

Comment:
Instead of itemizing all the examples in the readme, I think we should link to the example index, like [this](https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/keyvault/azkeys/README.md#examples). That index does the same thing and is easier to maintain.

Reply:
changed in [ff5948b](https://github.com/Azure/azure-sdk-for-go/pull/19946/commits/ff5948b90ad31665c18a3746e87a933290ae7449)